sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -47,16 +47,16 @@ exporter = AzureMonitorSpanExporter(
 # Regular open telemetry usage from here, see https://github.com/open-telemetry/opentelemetry-python
 # for details
 from opentelemetry import trace
-from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace import TracerSource
 from opentelemetry.sdk.trace.export import ConsoleSpanExporter
 from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
 
 # Simple console exporter
 exporter = ConsoleSpanExporter()
 
-trace.set_preferred_tracer_implementation(lambda T: Tracer())
-tracer = trace.tracer()
-tracer.add_span_processor(
+trace.set_preferred_tracer_implementation(lambda T: TracerSource())
```

Comment:
Why is the official sample calling it TracerProvider, and not TracerSource?
https://github.com/open-telemetry/opentelemetry-python#tracing

Reply:
Ok, they changed it already for future 0.5 (sigh)